docker-hub/api/latest.yaml

```diff
@@ -875,6 +880,267 @@ paths:
+
+  /v2/scim/2.0/ServiceProviderConfig:
+    get:
+      summary: Get service provider config
+      description: |
+        Returns a service provider config for Docker's configuration.
+      tags: [ scim ]
+      security:
+        - bearerSCIMAuth: [ ]
+      responses:
+        200:
+          $ref: '#/components/responses/scim_get_service_provider_config_resp'
+        401:
+          $ref: "#/components/responses/scim_unauthorized"
+        500:
+          $ref: "#/components/responses/scim_error"
+
+  /v2/scim/2.0/ResourceTypes:
+    get:
+      summary: List resource types
+      description: |
+        Returns all resource types supported for the SCIM configuration.
+      tags: [ scim ]
+      security:
+        - bearerSCIMAuth: [ ]
+      responses:
+        200:
+          $ref: "#/components/responses/scim_get_resource_types_resp"
+        401:
+          $ref: "#/components/responses/scim_unauthorized"
+        500:
+          $ref: "#/components/responses/scim_error"
+
+  /v2/scim/2.0/ResourceTypes/{name}:
+    get:
+      summary: Get a resource type
+      description: |
+        Returns a resource type by name.
+      tags: [ scim ]
+      parameters:
+        - name: name
+          in: path
+          schema:
+            type: string
+          example: User
+      security:
+        - bearerSCIMAuth: [ ]
+      responses:
+        200:
+          $ref: "#/components/responses/scim_get_resource_type_resp"
+        401:
+          $ref: "#/components/responses/scim_unauthorized"
+        404:
+          $ref: "#/components/responses/scim_not_found"
+        500:
+          $ref: "#/components/responses/scim_error"
+
+  /v2/scim/2.0/Schemas:
+    get:
+      summary: List schemas
+      description: |
+        Returns all schemas supported for the SCIM configuration.
+      tags: [ scim ]
+      security:
+        - bearerSCIMAuth: [ ]
+      responses:
+        200:
+          $ref: "#/components/responses/scim_get_schemas_resp"
+        401:
+          $ref: "#/components/responses/scim_unauthorized"
+        500:
+          $ref: "#/components/responses/scim_error"
+
+  /v2/scim/2.0/Schemas/{id}:
+    get:
+      summary: Get a schema
+      description: |
+        Returns a schema by ID.
+      tags: [ scim ]
+      parameters:
+        - name: id
+          in: path
+          schema:
+            type: string
+          example: urn:ietf:params:scim:schemas:core:2.0:User
+      security:
+        - bearerSCIMAuth: [ ]
+      responses:
+        200:
+          $ref: "#/components/responses/scim_get_schema_resp"
+        401:
+          $ref: "#/components/responses/scim_unauthorized"
+        404:
+          $ref: "#/components/responses/scim_not_found"
+        500:
+          $ref: "#/components/responses/scim_error"
+
+  /v2/scim/2.0/Users:
+    get:
+      summary: List users
+      description: |
+        Returns paginated users for an organization. Use `startIndex` and
```

Comment:
List users, returns paginated users for an organization. Use startIndex and count query parameters to receive paginated results.